src/content/en/2020/cms.md

```diff
@@ -23,3 +22,364 @@ published: 2020-12-09T00:00:00.000Z
+  sheets_gid="1074693681",
+  sql_file="distribution_of_page_weight_requests_and_co2_grams_per_cms_web_page.sql"
+  )
+}}
+
+CMS often load third party resources, such as external images, videos, scripts, or stylesheets:
+
+{{ figure_markup(
+  image="cms-third-party-bytes.png",
+  caption="Third party bytes.",
+  description="Bar chart showing the amount of third party bytes (in KB) for each percentile for CMS pages. At the 10th percentile it is 68.03 KB for desktop and 52.97 for mobile, at the 25th percentile it is 179.97 KB for mobile and 147.52 KB for mobile, at the 50% it's 436.44 KB for desktop and 397.33 MB for mobile, at the 75th percentile 936.18 KB for desktop and 927.64 for mobile and at the 90th percentile it is 1,735.25 KB for desktop and 1,766.15 for mobile.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vTkUxfuK-FCn3_IYDJiEsKDmdmyKb0TSEhG8dFc0XBIXej1NO2uUOmd-9NCbWuh-MZ3xzMhK_kNT-4u/pubchart?oid=1932733912&format=interactive",
+  sheets_gid="1916130433",
+  sql_file="third_party_bytes_and_requests_on_cmss.sql"
+  )
+}}
+
+We find that the median desktop CMS page has 27 third party requests with 436 KB of content, with the mobile equivalent generating 26 requests with 397 KB of content.
+
+One of the main ways a CMS can influence its page load size is by supporting and encouraging the usage of more efficient formats. Images are behind only video in their contribution to page weight. 
+
+{{ figure_markup(
+  image="cms-median-cms-kb-per-resource-type.png",
+  caption="Median CMS KB per resource type.",
+  description="Bar chart showing the median resource size in KB of each resource type for CMSs. Audio 14 KB on desktop and 10 KB on mobile, CSS is 102 KB on desktop and 98 on mobile, Fonts are 152 KB on desktop and 125 on mobile, Images are 1,273 KB on desktop and 1,280 on mobile, Video is 1,665 KB on desktop and 2,139 KB on mobile and XML is 1 KB for both desktop and mobile.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vTkUxfuK-FCn3_IYDJiEsKDmdmyKb0TSEhG8dFc0XBIXej1NO2uUOmd-9NCbWuh-MZ3xzMhK_kNT-4u/pubchart?oid=74536341&format=interactive",
+  sheets_gid="1846606749",
+  sql_file="third_party_bytes_and_requests_on_cmss.sql"
+  )
+}}
+
+Video contributes a larger percentage per resource type here. Making video more efficient is an interesting area for future research. Here our focus is on images. Popular image formats are JPG, PNG, GIF, SVG, WebP, and ICO. Of these, [WebP is the most efficient in most situations](https://developers.google.com/speed/webp/), with WebP lossless images [26% smaller](https://developers.google.com/speed/webp/docs/webp_lossless_alpha_study#results) than equivalent PNGs and [25-34% smaller](https://developers.google.com/speed/webp/docs/webp_study) than comparable JPGs. We see, however, that WebP is the second least popular image format across all CMS pages:
```

Comment:
I think video stands out because of autoplay. The test will load about 30 seconds of the video stream before timing out. Making videos more efficient may not really help much, but stopping autoplay might. Video is also relatively rare compared to images.

Images are an interesting thing to focus on, and I think efficiency makes more sense here. There's also lazy loading as a potential optimization. @felixarntz has done some work here related to native image lazy loading in WordPress core and may be able to add additional context.

Reply:
that's great insight, thanks! Added a bit about autoplay and lazy loading, but not mentioned it's in WordPress Core.